src/DetailsView/components/assessment-instance-table.tsx

```diff
@@ -137,7 +137,7 @@ export class AssessmentInstanceTable extends React.Component<AssessmentInstanceT
     private isAnyInstanceStatusUnknown(items: InstanceTableRow[], step: string): boolean {
         return items.some(
             item =>
-                has(item.instance.testStepResults, step) &&
+                hasIn(item.instance.testStepResults, step) &&
```

Comment:
Added hasIn for deep comparison as lint was throwing error items can be undifined